### DISCUSSION TITLE
Individual entries per table, per column?

### DISCUSSION THREAD

**Discussion by cpryor81** [entechlog/dbt-snow-mask#49] [Q&A] [answered]
I'm trying to find out if there's a shorthand approach to masking tables/columns.  Currently, I have 34 tables and over 400 columns.  Am I correct to assume that I'll need to set up (in the yaml) something similar to what's below?  This is fine, I just want to make sure there's no better way to do this, before I create all the entries.  

- name: Table1 
  columns: 
  - name: Field1
     meta: 
       masking_policy: encrypt_pii
  - name: Field2
     meta: 
      masking_policy: encrypt_pii
- name: Table2 
  columns: 
  - name: Field1
     meta: 
       masking_policy: encrypt_pii
  - name: Field2
     meta: 
      masking_policy: encrypt_pii
- name: Table3
  columns: 
  - name: Field1
     meta: 
       masking_policy: encrypt_pii
  - name: Field2
     meta: 
      masking_policy: encrypt_pii
.
.
.

**Comment by entechlog** [comment, marked as answer]
Hello @cpryor81, yes, unfortunately that is the only way now. You can generate the source yml of an existing table using the below command, then add the meta part if that makes it a little easier. Let me know if you can think of any ideas to enhance to cover the multiple column scenario.

`dbt run-operation generate_model_yaml --args '{"model_name": "dim_date"}'
`
